utils/RunONNXModel.py

```diff
@@ -296,15 +297,17 @@ def generate_random_input(input_signature, input_shapes):
         if (np.issubdtype(np_elem_type, np.floating)):
             lb = -1.0
             ub = 1.0
-
-        elif (np.issubdtype(np_elem_type, np.integer)):
-            lb = -10
+        elif (np.issubdtype(np_elem_type, np.uint8)):
```

Comment:
NIT: is it possible to use a non-sized type (like for integers, its `np.integer`. Maybe there is a `np.unsigned`.. or if not, we could use the sizes 8, 16, 32, 64.

Reply:
I seems there is no `np.unsigned`, so I checked them case by case. Also, I added two new options `--lower-bound` and `--upper-bound` to allow users set the range for random inputs.